src/components/Authentication/ForgotPassword.js

```diff
@@ -48,34 +69,74 @@ const ForgotPassword = () => {
     }
 
     return (
-        <Paper className={classes.root}>
-            <Typography align="left" className={classes.header} color="primary">
-                {submitted ? "reset email sent!" : "forgot your password?"}
-            </Typography>
-            <Typography align="left" className={classes.info}>
-                {submitted ?
-                    "Follow the instructions on the email to reset your password." :
-                    "Enter the email for your account and we will send you an email link to reset your password:"}
-            </Typography>
-            {submitted ?
-                <Button className={classes.primaryButton} color="primary"
-                    component={Link} data-cy="return" to={{
-                        "pathname": "/login",
-                        "state": {email},
-                    }} variant="contained">
-                    Back to login
-                </Button> :
-                <form onSubmit={handleSubmit}>
-                    <TextField fullWidth inputProps={{"data-cy": "emailField"}}
-                        label="E-Mail" margin="normal"
-                        onChange={handleEmailInput} value={email} />
-                    <Button className={classes.primaryButton} color="primary"
-                        data-cy="reset" disabled={!email}
-                        type="submit" variant="contained">
-                        Send Reset Email
-                    </Button>
-                </form>}
-        </Paper>
+        <div>
+            <Ellipse1 className="ellipse1" />
+            <Ellipse2 className="ellipse2" />
+            <Picture1 className="picture1" />
+            <div className="logo">
+                <Typography className="title">
+                    omou
+                </Typography>
+            </div>
+            <form className="Login">
+                <Grid className="resetPassword" container>
+                    <Grid item md={6} />
+                    <Grid item md={6}>
```

Comment:
Instead of doing all of the `request ? ... : ...` to do the next step, create a new component to organize it better then toggle between the two components.